### PR TITLE
Fix incorrect image paths

### LINKED ISSUE #256: Images in UG not rendering properly
E.g.
Screenshot 2024-11-07 at 1 53 14 PM

### PR DESCRIPTION
**Pull request by zackjh**
Resolves #256

Allows images in UG to render properly.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This project is built on top of [AddressBook-Level3 Project](https://se-educatio
    Note: A GUI similar to the image below should appear in a few seconds. The image below consist of existing sample data.<br>
    </box>
 
-   ![defaultUi](/images/uiWithDescription.png)
+   ![defaultUi](images/uiWithDescription.png)
```
Comment (bryanckh) [inline docs/UserGuide.md:42]: Good job spotting and fixing the bug on the image path!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -226,7 +226,7 @@ Format: `radd CLIENT_INDEX a/ADDRESS [s/RENTAL_START_DATE] [e/RENTAL_END_DATE] [
 Displays a list of all clients within TrueRental in a table format.
 
 Format: `list`
-![list](/images/listUi.png)
+![list](images/listUi.png)
```
Comment (bryanckh) [inline docs/UserGuide.md:229]: And also for this image path too!

**Review by bryanckh — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (winstonlimjy) [reply docs/UserGuide.md:42]: Nice thanks for the change!

File: docs/UserGuide.md (hunk 2 above)
Comment (winstonlimjy) [reply docs/UserGuide.md:229]: Nice thanks for the change!

**Pull request merged**